src/reducers/systemIntakeReducer.ts

```diff
@@ -1,14 +1,22 @@
 import { STORE_SYSTEM_INTAKE } from '../constants/actions';
-import { SystemIntakeState } from '../types/systemIntake';
+import { SystemIntakeState, SystemIntakeForm } from '../types/systemIntake';
+import {
+  initialSystemIntakeForm,
+  prepareSystemIntakeForApp
+} from '../data/systemIntake';
 
 const initialState: SystemIntakeState = {};
 
 function systemIntakeReducer(state = initialState, action: any): any {
+  let systemIntake: SystemIntakeForm = initialSystemIntakeForm;
```

Comment:
lines 11-14 should be able to be removed after the changes to this file are made.